### DISCUSSION TITLE
Abstruct Group operation

### DISCUSSION THREAD

**Discussion by kylinholmes** [PowerShell/PowerShell#16581] [Enhancements]
### Summary of the new feature / enhancement

When I look up command about group,like this:
![image](https://user-images.githubusercontent.com/45586871/145143157-e53a3c71-142a-4331-894e-831b3f4cadd9.png)
There are many command. 

And, if it be a possible to **abstruct these group command into filesystem(or Drive?)** like Env,HKCU,Cert in Get-PSDrive, where I could operator these abstruct object easily by new-item, set-item, remove-item,get-item
![image](https://user-images.githubusercontent.com/45586871/145144034-88c83aad-3816-48c7-afec-d688a4bdf55c.png)

### Proposed technical implementation details (optional)

It may like these (omit some group and users):
![image](https://user-images.githubusercontent.com/45586871/145144997-6851cb23-b01e-46aa-bd89-a9412be74eb0.png)

If there are more group like **"remote group"**, I cloud also add them into this **"Group dir"**,rather than write another set of cmdlets called Get/Set/New/Remove - RemoteGroup(Member).

More over, plz add more Drive into PSDrive, It's Really helpful.

**Comment by iSazonov** [comment]
You could create a custom format view as you need.

**Reply by kylinholmes** [reply]
Alright,I'll try it.

**Comment by daxian-dbw** [comment]
@kylinholmes What you are asking is a custom provider that organizes users, similar to the `Cert:`  and `WSMan:`. This can be done as a module by community members who are interested in this idea, but I don't think PowerShell team will have the capacity to do it.

I will move this issue to Discussions, where you may find someone also likes this idea.

**Reply by kylinholmes** [reply]
Thanks,I'll tring to do it. I just find a doc. But you know the official is better.

**Comment by daxian-dbw** [comment]
You may be interested in looking into https://github.com/PowerShell/SHiPS, which is a module that speeds up your work on a PowerShell provider.

**Reply by kylinholmes** [reply]
OK，thanks
